src/core/component.js

```diff
@@ -177,8 +178,10 @@ Component.prototype = {
    */
   flushToDOM: function () {
     var attrValue = this.attrValue;
+    var attrName;
     if (!attrValue) { return; }
-    HTMLElement.prototype.setAttribute.call(this.el, this.name, this.stringify(attrValue));
+    attrName = this.name + (this.id ? '__' + this.id : '');
```

Comment:
Starting to seem more useful to store the entire attribute name rather than the raw ID.